paper-api/src/main/java/org/bukkit/entity/Player.java

```diff
@@ -3275,6 +3275,22 @@ default boolean hasResourcePack() {
      */
     java.util.Locale locale();
     // Paper end
+
+    /**
+     * Returns the locale the server will use to send messages translated via the Adventure global translator.
+     * By default, the value of {@link Player#locale()} is used.
+     *
+     * @return the locale.
+     */
+    java.util.Locale getEffectiveLocale();
```

Comment:
We have hardforked and can now import things in non-patched sources like the API :partying_face: